src/main/java/com/github/technus/tectech/thing/metaTileEntity/multi/GT_MetaTileEntity_EM_dataBank.java

```diff
@@ -149,13 +153,21 @@ protected CheckRecipeResult checkProcessing_EM() {
             mMaxProgresstime = 20;
             mEfficiencyIncrease = 10000;
             return SimpleCheckRecipeResult.ofSuccess("providing_data");
+        } else if (eDataAccessHatches.size() > 0 && eWirelessDataOutputs.size() > 0) {
```

Comment:
Merge with the if above if the bodies are the same `(eStacksDataOutputs.size() > 0 || eWirelessDataOutputs.size() > 0)`

Reply:
Doing this eliminates the ability to change the power consumption cost,efficiency,etc of the wireless hatch unless there is a nested if within the (eStacksDataOutputs.size() > 0 || eWirelessDataOutputs.size() > 0) explicitltly checking for the eWirelessDataOutputs and adjusting appropriately. Once again, leaving that up for debate on what is appropriate there.